apps/api/src/jobs/consumers/summary/summary.consumer.ts

```diff
@@ -0,0 +1,60 @@
+import { Logger } from '@nestjs/common';
+import { Process, Processor } from '@nestjs/bull';
+import { Job } from 'bull';
+import { InjectRepository } from '@nestjs/typeorm';
+import { SUMMARY_QUEUE } from 'src/modules/summary/queues/summary.queue';
+import { SummaryService } from 'src/modules/summary/summary.service';
+import { Repository } from 'typeorm';
+import { Summary } from 'src/modules/summary/entities/summary.entity'; 
+import { JobStatus } from 'src/common/constants/summary';
+
+@Processor(SUMMARY_QUEUE)
+export class SummaryConsumer {
+  private readonly logger = new Logger(SummaryConsumer.name);
+
+  constructor(
+    @InjectRepository(Summary)
+    private readonly summaryRepository: Repository<Summary>,
+    private readonly summaryService: SummaryService, 
+    // private readonly summaryConsumer: SummaryConsumer
```

Comment:
Commented code not removed. Please check your PRs.

Reply:
Removed